lambdas/handlers/lloyd_george_record_stitch_handler.py

```diff
@@ -30,166 +18,16 @@
 @set_request_context_for_logging
 @validate_patient_id
 @override_error_check
+@handle_lambda_exceptions
 @ensure_environment_variables(
     names=["LLOYD_GEORGE_DYNAMODB_NAME", "LLOYD_GEORGE_BUCKET_NAME"]
 )
-def lambda_handler(event, context):
+def lambda_handler(event, _context):
```

Comment:
Any reason for this change from `context` to `_context`?

Reply:
Reverted it to `context` :)

Reply:
Ah no worries, should be fine in that case just thought I'd bring it up in case it was accidental